pkgs/servers/lenpaste/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, buildGoModule, fetchFromGitea }:
+
+buildGoModule rec {
+  pname = "lenpaste";
+  version = "1.3";
+
+  src = fetchFromGitea {
+    domain = "git.lcomrade.su";
+    owner = "root";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-d+FjfEbInlxUllWIoVLwQRdRWjxBLTpNHYn+oYU3fBc=";
+  };
+
+  vendorSha256 = "sha256-PL0dysBn1+1BpZWFW/EUFJtqkabt+XN00YkAz8Yf2LQ=";
+
+  ldflags = [
+    "-w"
+    "-s"
+    "-X main.Version=${version}"
+  ];
+
+  subPackages = [ "cmd" ];
+
+  postInstall = ''
+    mv $out/bin/cmd $out/bin/lenpaste
+  '';
+
+  meta = with lib; {
+    description = "Open source analogue of pastebin.com. There is its own API.";
```

```suggestion
    description = "A web service that allows you to share notes anonymously, an alternative to pastebin.com";
```

See https://nixos.org/manual/nixpkgs/unstable/#var-meta-description
and "`meta.description` should" in https://github.com/NixOS/nixpkgs/blob/8d0598da60f5e81748da095cc5eefc31d6e59ec3/CONTRIBUTING.md#submitting-changes